plugins/phoenix/phoenix_main.ml

```diff
@@ -76,10 +62,23 @@ let man = [
       target file will be used as a directory name."
 ]
 
-let info = Term.info ~man ~doc ~version name
-
 let () =
-  match Term.eval ~argv ~catch:false (options,info) with
-  | `Ok options -> Project.register_pass' (run options)
-  | `Version | `Help -> exit 0
-  | `Error _ -> exit 1
+  Config.manpage man;
+  Config.when_ready (fun {Config.get=(!)} ->
+      let cfg_format =
+        let self = [] in
+        let self = if !with_name then `with_name :: self else self in
+        let self = if !with_asm  then `with_asm  :: self else self in
+        let self = if !with_bil  then `with_bil  :: self else self in
+        match self with
+        | [] -> [`with_name]
+        | _ as x -> x in
```

Comment:
it is not the choice of one from three, it should work as a checkbox basically. Something like this:
```ocaml
let cfg_format = List.filter [`with_name,!with_name; `with_asm, !with_asm; `with_bil, !with_bil] ~f:snd |> List.map ~f:fst
```